docs/api/exchange_rates.rst

```diff
@@ -0,0 +1,455 @@
+Exchange Rates API
+==================
+
+These endpoints will allow you to easily manage exchange rates. Base URI is `/api/v1/exchange-rates`.
```

Comment:
Would be nice to have the base uri somehow highlighted, either `note` or just plain old next line.
It's the vital part which most people will look for.